OpenSim/Tools/MarkerPlacer.cpp

```diff
@@ -261,7 +261,7 @@ bool MarkerPlacer::processModel(Model* aModel,
                                                    _timeRange[1]);
     for(size_t r = staticPoseTable.getNumRows() - 1; r > 0; --r)
         staticPoseTable.removeRowAtIndex(r);
-    staticPoseTable.updRowAtIndex(0) = avgRow;
+    staticPoseTable.appendRow(_timeRange[0], avgRow);
```

Comment:
I think the table will have 2 rows, but I think it's supposed to have only 1 row. The for-loop removes all but the first row, and then you append a row.